pages/linux/edquota.md

```diff
@@ -0,0 +1,28 @@
+# edquota
+
+> Edit quota for a user or group. By default it operates on all file systems with quotas.
+> Quota information is stored permanently in the quota.user and quota.group files in the root of the file system.
+
+- Edit quota of current user (default):
+
+`edquota --user {{user}}`
```

Comment:
I see, sorry about that. The tool will not run without any parameters. I believe it is correct as it is now.